marathon-api/cli.py

```diff
@@ -1,37 +1,55 @@
 #!/usr/bin/env python
 
-import krux.logging
-import sys
-import socket
-import json
-from pprint import pprint
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+
+#
+# Third party libraries
+#
+
 from marathon import MarathonClient
 from marathon.models import MarathonApp
+
+#
+# Internal libraries
+#
+
+import krux.logging
 from krux.cli import Application, get_group, get_parser
+import marathonapi
 
 
-class MarathonClientApp(Application):
+class Marathon_cli_app(Application):
```

Comment:
nit: MarathonClientApp is a better name for classes (classes don't usually have underscores in them)